scripts/primir_quantification.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env python
+
+"""Tabulate bedtools intersect output BED file.
+
+Read the input BED file, calculate the sum of the intersecting contributions
+for each feature and print the result in tab-delimited format. The name of the
+feature is determined by the value of the --id argument, which must match one
+of the fields in the attributes column of the original GFF/GTF file used in
+the bedtools intersect command. If a SAM file is provided, the contribution
+of each alignment is computed as 1/NH.If an alignment is not found, its
+contribution is assumed to be 0. If no SAM file is provided, the contribution
+of each alignment is assumed to be 1. If either the BED or SAM file is empty,
+no output is produced. The output columns are also determined by the flags --read-ids
+and --feat-extension. If --read-ids is set, the last column will contain a
```

Comment:
From the description, the column order is not clear in the case that both flags are set. Will the IDs still be in the last column, or will the extensions be added?

If you resolve this you can merge, no need to review again.